revdep/problems.md

```diff
@@ -0,0 +1,17881 @@
+* Date/Publication: 2025-01-07 13:00:02 UTC
+* Number of recursive dependencies: 130
+
+Run `revdepcheck::cloud_details(, "actxps")` for more info
+
+</details>
+
+## Newly broken
+
+*   checking re-building of vignette outputs ... ERROR
+    ```
+    Error(s) in re-building vignettes:
+    --- re-building ‘actxps.Rmd’ using rmarkdown
+    Warning: thematic was unable to resolve `bg='auto'`. Try providing an actual color (or `NA`) to the `bg` argument of `thematic_on()`. By the way, 'auto' is only officially supported in `shiny::renderPlot()`, some rmarkdown scenarios (specifically, `html_document()` with `theme!=NULL`), in RStudio, or if `auto_config_set()` is used.
+    Warning: thematic was unable to resolve `fg='auto'`. Try providing an actual color (or `NA`) to the `fg` argument of `thematic_on()`. By the way, 'auto' is only officially supported in `shiny::renderPlot()`, some rmarkdown scenarios (specifically, `html_document()` with `theme!=NULL`), in RStudio, or if `auto_config_set()` is used.
+    Warning: thematic was unable to resolve `accent='auto'`. Try providing an actual color (or `NA`) to the `accent` argument of `thematic_on()`. By the way, 'auto' is only officially supported in `shiny::renderPlot()`, some rmarkdown scenarios (specifically, `html_document()` with `theme!=NULL`), in RStudio, or if `auto_config_set()` is used.
+    
+    Quitting from actxps.Rmd:128-130 [plot]
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    <error/rlang_error>
+    Error in `adjust_color()`:
+    ...
+    
+    Error: processing vignette 'transactions.Rmd' failed with diagnostics:
+    Internal error: adjust_color() expects an input of length 1
+    --- failed re-building ‘transactions.Rmd’
+    
+    SUMMARY: processing the following files failed:
+      ‘actxps.Rmd’ ‘misc.Rmd’ ‘transactions.Rmd’
+    
+    Error: Vignette re-building failed.
+    Execution halted
+    ```
+
+# adklakedata
+
+<details>
+
+* Version: 0.6.1
+* GitHub: https://github.com/lawinslow/adklakedata
+* Source code: https://github.com/cran/adklakedata
+* Date/Publication: 2018-02-16 19:08:16 UTC
+* Number of recursive dependencies: 62
+
+Run `revdepcheck::cloud_details(, "adklakedata")` for more info
+
+</details>
+
+## Newly broken
+
+*   checking examples ... ERROR
+    ```
+    Running examples in ‘adklakedata-Ex.R’ failed
+    The error most likely occurred in:
+    
+    > ### Name: adk_lake_shapes
+    > ### Title: Return path to Lake Polygons Shapefile
+    > ### Aliases: adk_lake_shapes
+    > 
+    > ### ** Examples
+    > 
+    > library(sf)
+    Linking to GEOS 3.12.1, GDAL 3.8.4, PROJ 9.4.0; sf_use_s2() is TRUE
+    > bl = read_sf(adklakedata::adk_shape())
+    Error in process_cpl_read_ogr(x, quiet, check_ring_dir = check_ring_dir,  : 
+      package tibble not available: install first?
+    Calls: read_sf ... st_read -> st_read.character -> process_cpl_read_ogr
+    Execution halted
```

Comment:
We moved tibble to suggests (#5990), so it isn't automatically downloaded for reverse dependencies. This should be solved at their end by setting a tibble dependency, I think.